src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
+$main-color: #060b35;
+$secondary-color: #616070;
+$white-color: #fff;
+$lightblue-color: #00acdc;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+  color: $main-color;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__product-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
```

Comment:
line-height: 18px is duplicated in a few places, please move all duplicated css code to variables